source/draft_auth.rst

```diff
@@ -33,6 +33,30 @@ fields as necessary for that kind of connection.
 
 These other fields might be things like ``password``, ``token``, ``key``, etc.
 
+Token-based authentication
+--------------------------
+
+Token-based is the primary and default authentication method for TimeSync. TimeSync
+provides an endpoint at ``/login``, which should be sent a POST request, with the body
+fitting one of the other available authentication schemes (e.g. Password, LDAP). The
+endpoint will then return a JWT token string as its response body. This response body
+will be used to access the other endpoints:
+
+POST endpoints:
+
+.. code-block:: javascript
+
+    {
+      "auth": {
+        "type": "token",
+        "token": ...
+      },
+      ...
```

Comment:
You should add a little more context or a complete example below (hitting /times/:some_uuid you would craft this JSON body...).